scripts/EGAP/files_to_import_structure.py

```diff
@@ -0,0 +1,55 @@
+import os
+import shutil
+import argparse
+from distutils.dir_util import copy_tree
+
+get_item_id = lambda _path: _path.split(os.sep)[-1].split(' ')[0]
+
+parser = argparse.ArgumentParser()
+parser.add_argument('-source', '--source', help='')
```

Comment:
Could you fill in the help text with what these arguments are expecting? Not just that they're paths, but what the paths represent.